src/zcbor_encode.c

```diff
@@ -395,9 +397,9 @@ bool double_put(zcbor_state_t *state, double input)
 }
 
 
-bool zcbor_any_encode(zcbor_state_t *state, void *input)
+bool zcbor_any_encode(zcbor_state_t *state, const void *unused)
 {
-	return zcbor_nil_put(state, input);
+	return zcbor_nil_put(state, NULL);
```

Comment:
What is the point of this function now that it acts as a wrapper for `zcbor_nil_put` ?

Reply:
Removed. It was used in code generation, so I changed zcbor.py to use `zcbor_nil_put` directly.